src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/WebClientConfiguration.kt

```diff
@@ -29,6 +37,18 @@ class WebClientConfiguration(
     return getWebClient(hmppsAuthUrl, oauth2Client)
   }
 
+  @Bean
+  fun visitSchedulerWebClient(authorizedClientManager: OAuth2AuthorizedClientManager): WebClient {
```

Comment:
getVisitSchedulerWebClient or createVisitSchedulerWebClient

Reply:
that is also the bean name which is used by qualifier annotation in VisitSchedulerClient so I think visitSchedulerWebClient workd better and it is also consistent with what we have in the Orchestration service.